pyrk/inp/sim_info.py

```diff
@@ -32,7 +32,7 @@ def __init__(self,
         self.rho_ext = self.init_rho_ext(rho_ext)
         self.feedback = feedback
         self.ne = self.init_ne()
-        self.th = th_system.THSystem(kappa=kappa, components=components)
+        self.th = th_system.THSystemSphFVM(kappa=kappa, components=components)
```

Comment:
The user shouldn't have to specify it. The spherical version should only be used if the components list (specified by the user) includes one or more components which are of "sph" type. Right?

We should talk about it at some point. I'm back on thursday.

In the way that it's currently implemented, with both (and, in fact, three if you include the PS type) classes existing but only one being used, there is definitely something that will need to be cleaned up. 

As far as documentation goes, that's why the docstrings end up in the api documentation online. If you think it needs more description, that's always a good thing to add, but I think your docstring for this class is currently good as it is.